### PR TITLE
Adding phenoshift recipe

### PR DESCRIPTION
**Pull request by iimpulse**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x]  License file is packaged (see here for an example)
- [x] Source is from official source
- [x]  Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x]  If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (url) rather than a repo (e.g. git_url) is used in your recipe (see here for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/phenoshift/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "phenoshift" %}
+{% set version = "0.0.4.dev0" %}
```
Comment (scopatz) [inline recipes/phenoshift/meta.yaml:2]: Please release full versions. This seems ambiquous if it is a dev version or not

File: recipes/phenoshift/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f451ffc36ce85649b99e64ecd56b8b26e7f10a6587556b1b9eb2670f18d1d96e
+
+build:
+  number: 0
```
Comment (scopatz) [inline recipes/phenoshift/meta.yaml:13]: This should be a noarch python package

File: recipes/phenoshift/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
```
Comment (scopatz) [inline recipes/phenoshift/meta.yaml:21]: You probably don't need setuptools since you have pip

File: recipes/phenoshift/meta.yaml (hunk 2 above)
Comment (iimpulse) [reply recipes/phenoshift/meta.yaml:14]: Agreed, I had this before but the linter doesn't like it. Is the way I am writing it incorrect?

**Pull request closed without merging**